consensus/vbft/chain_store.go

```diff
@@ -24,19 +24,26 @@ import (
 	"github.com/ontio/ontology/common"
 	"github.com/ontio/ontology/common/log"
 	"github.com/ontio/ontology/core/ledger"
+	"github.com/ontio/ontology/core/store"
+	"github.com/ontio/ontology/core/store/overlaydb"
+	"github.com/ontio/ontology/events/message"
 )
 
 type ChainStore struct {
 	db              *ledger.Ledger
 	chainedBlockNum uint32
 	pendingBlocks   map[uint32]*Block
+	execResult      *store.ExecuteResult
```

Comment:
bind ExecuteResult with pendingBlocks.

type PendingBlock { *Block, *store.ExecuteResult }
change pendingBlock to map[uint32]PendingBlock

In the following Get/SetExecuteMerkleRoot() to Get/SetExecuteMerkleRoot(blockNum), 
and Get/Set ExecWriteSet

Reply:
has fixed